docker-app/qfieldcloud/subscription/admin.py

```diff
@@ -48,6 +51,24 @@ def queryset(self, request, queryset):
         return queryset
 
 
+class ActiveUntilFilter(admin.SimpleListFilter):
+    title = "expiration date"
+    parameter_name = "active_until"
+
+    def lookups(self, request, model_admin):
+        return [
+            ("1", "Next week"),
+            ("2", "Next 2 weeks"),
+            ("4", "Next 4 weeks"),
+        ]
+
+    def queryset(self, request, queryset):
+        if self.value():
+            now = timezone.now()
+            future_date = now + timedelta(weeks=int(self.value()))
+            return queryset.filter(active_until__range=(now, future_date))
```

Comment:
Quite smart. I would have done `active_until__gt=now(), active_since__lt=future_date`, but yours is more elegant, thanks!